packages/react/test/specs/components/Dropdown/test-utils.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import Dropdown, { DropdownProps } from 'src/components/Dropdown/Dropdown'
+import DropdownSearchInput from 'src/components/Dropdown/DropdownSearchInput'
+import { findIntrinsicElement, mountWithProvider } from 'test/utils'
+
+const items = ['item0', 'item1', 'item2', 'item3', 'item4', 'item5']
+
+const renderDropdown = (props: DropdownProps = {}) => {
```

Comment:
I prefer having all the component interaction abstracted in the tests, and have all the getting / action logic separately.